### ISSUE TITLE
Exporting to yolo when using local files failes

### ISSUE THREAD

**Issue by magicaltoast** [heartexlabs/label-studio#1928]
**Describe the bug**
When exporting annotation to yolo format while using local storage, the image folder is empty.
It's impossible that label studio can't access these files because I can see them normally in UI
Log from terminal
```
2022-01-18 04:11:15,239] [label_studio_converter.converter::convert_to_yolo::582] [ERROR] Unable to download /data/local-files/?d=...rest_of_file_path
```
**To Reproduce**
1. Use local storage
2. Click export
3. Choose yolo format
4. Check label-studio logs for errors

**Expected behavior**
Images folder to be full of images

**Environment (please complete the following information):**
 - OS: Ubuntu 21.04
 - Label Studio Version: 1.4

**Comment by makseq** [comment]
@magicaltoast Please, check this thread: https://github.com/heartexlabs/label-studio/issues/1350#issuecomment-905050762

TLDR: We dropped copying of images to the export archive by security reasons, LS can't access user data directly, it can be done only on user side in the browser.

**Comment by makseq** [comment]
However we are thinking about integrating an export & converter functionality to SDK: this operation can be performed on the user side and it's secure.

**Comment by makseq** [comment]
Feel free to reopen if you have more question / issue on this topic.
